db/config.ts

```diff
@@ -1,7 +1,16 @@
 import { defineDb, defineTable, column } from 'astro:db';
+import { createId } from '@paralleldrive/cuid2';
 
 // https://astro.build/db/config
 
+const EmailLog = defineTable({
```

Comment:
I know I'm the one who suggested EmailLog but we might want to match the name with the intent of table. In this case we are only logging emails sent via the contact form.